### PR TITLE
0.4.30 Add integration tests, change response object naming, improve get_multitag_datapoints(), and fix minor issues.

### PR DESCRIPTION
**Pull request by erlendcognite**

### CODE REVIEW

File: cognite/cloud_storage.py [hunk 1]
```diff
@@ -88,7 +93,66 @@ def upload_file(file_name, file_path=None, directory=None, source=None, file_typ
...
+    api_key, project = config.get_config_variables(kwargs.get('api_key'), kwargs.get('project'))
+    url = config.get_base_url() + '/projects/{}/storage/{}'.format(project, id)
+    headers = {
+        'api-key': api_key,
+        'accept': 'application/json'
+    }
+    res = _utils.get_request(url=url, headers=headers, cookies=config.get_cookies())
+    if get_contents:
+        dl_link = res.json()['data']
+        res = requests.get(dl_link)
```
Comment (psalaberria002) [inline cognite/cloud_storage.py]: No try catch here? What if it fails to download?

File: tests/test_config.py [hunk 2]
```diff
@@ -25,6 +23,7 @@ def test_set_base_url():
     new_url = 'another_url'
     config.set_base_url(new_url)
     assert config.get_base_url() == new_url
+    config.set_base_url(None)
```
Comment (psalaberria002) [inline tests/test_config.py]: I am not a big fan of resetting stuff after running the test like this. Isn't there anything like setup/ tearDown similar to unittests?

File: cognite/assets.py [hunk 3]
```diff
@@ -26,7 +26,7 @@ def get_assets(name=None, path=None, description=None, metadata=None, depth=None
 
         metadata (str):         The metadata values used to filter the results.
 
-        depth (str):            Get sub assets up oto this many levels below the specified path.
+        depth (intq):            Get sub assets up oto this many levels below the specified path.
```
[the remark this answers is not in the document]
Comment (erlendcognite) [reply cognite/assets.py]: oups

**Comment by psalaberria002** [comment]
by the way, why do I get so many changes in this PR? many of them.were already merged in the previous PR. Are you doing something strange with your branches locally?

**Comment by psalaberria002** [comment]
Sorry, my bad. I got confused with the secret, I thought it was the codecov secret again. Forget my previous comment

**Pull request merged**